### PR TITLE
ES6: Awesome Books

### PR DESCRIPTION
**Pull request by TEMESGEN-G25**
## In this project I :

- Rewrote the Awesome Books projects & used ES6 syntax.
- Created a module directory.
- Kept all functionalities of the app without errors.
- Displayed the current date below the navigation bar using Luxon.

### CODE REVIEW

File: modules/library.js [hunk 1]
```diff
@@ -0,0 +1,108 @@
+const list = document.getElementById('book_list');
+const AddBook = document.getElementById('AddBook');
+
+export default class Library {
```
Comment (dasileker) [inline modules/library.js:4]: - [ ] Please use the ES6 arrow function.

File: modules/navigate.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    const link = document.getElementById(navigation.link);
+
+    link.addEventListener('click', (e) => {
+      e.preventDefault();
+
+      const sections = document.querySelectorAll('main > section');
+      sections.forEach((section) => {
+        section.classList.add('_display_f_');
+      });
+
```
Comment (dasileker) [inline modules/navigate.js:27]: - [ ] The navbar is not a requirement now, please focus on making the app work correctly.

**Review by dasileker — CHANGES_REQUESTED** [review]
### **Hi @NatiG25,** 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

- Your app is not working, we should be able to add and remove books but unfortunately, we can't, please fix those issues and request another review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: modules/library.js (hunk 1 above)
Comment (TEMESGEN-G25) [reply modules/library.js:4]: Thank you for that suggestion, but I don't understand what you want me to do exactly. How can I use the arrow function for declaring a class or a variable?

Comment (dasileker) [reply modules/library.js:4]: hello again, sorry to reply late, you see line number 4 you have a class library that's what I  want you to look at. you can use const instead.

Comment (TEMESGEN-G25) [reply modules/library.js:4]: I used const it resulted in an error. This is js class I don't see where I could use an arrow function.

Comment (NatiG25) [reply modules/library.js:4]: With all due respect, can you please check if this change is necessary?

**Review by mikethreels — APPROVED** [review]
Hi Nati, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**